cargo/src/cli/mod.rs

```diff
@@ -120,3 +120,180 @@ impl Display for Compression {
         write!(f, "{}", msg)
     }
 }
+
+#[derive(clap::Args, Debug, Clone)]
+pub struct Src {
```

Comment:
I think this needs (hidden) aliases for SrcTar and SrcDir so that existing _service files "just work" when we upgrade.

Reply:
True. I will do it in a bit

Reply:
No rush :)